<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt VTK Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sagittal plane</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QVTKRenderWidget" name="view2" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Axial plane</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="SceneWidget" name="sceneWidget"/>
    </item>
    <item row="1" column="0">
     <widget class="QVTKRenderWidget" name="view1" native="true"/>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Coronal plane</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>VTK File</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QVTKRenderWidget" name="view3" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionOpen_folder"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile"/>
   <addaction name="actionOpen_folder"/>
   <addaction name="actionZoomToExtent"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/open_file.png</normaloff>:/icons/open_file.png</iconset>
   </property>
   <property name="text">
    <string>Open VTK</string>
   </property>
   <property name="toolTip">
    <string>Open file...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionZoomToExtent">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/zoom_to.png</normaloff>:/icons/zoom_to.png</iconset>
   </property>
   <property name="text">
    <string>Zoom to extent</string>
   </property>
   <property name="toolTip">
    <string>Zoom to extent</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/quit.png</normaloff>:/icons/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOpen_folder">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/open_folder.png</normaloff>:/icons/open_folder.png</iconset>
   </property>
   <property name="text">
    <string>Open DICOM</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SceneWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>scenewidget.h</header>
   <slots>
    <slot>zoomToExtent()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>QVTKRenderWidget</class>
   <extends>QWidget</extends>
   <header>QVTKRenderWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionOpenFile</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showOpenFileDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomToExtent</sender>
   <signal>triggered()</signal>
   <receiver>sceneWidget</receiver>
   <slot>zoomToExtent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAboutDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showOpenFileDialog()</slot>
  <slot>showAboutDialog()</slot>
 </slots>
</ui>
